File: benchmark/corpus/18_correct_but_refused/a_count_until_end.js
// The control: counts the lines of its input, reading until the input ends. Correct, and unbounded by design.
const reader = Deno.stdin.readable.getReader();
let text = "";
let done = false;
while (!done) {
  const chunk = await reader.read();
  done = chunk.done;
  if (chunk.value) text += new TextDecoder().decode(chunk.value);
}
console.log(text.split("\n").filter((l) => l !== "").length + " lines");

File: benchmark/corpus/18_correct_but_refused/a_count_until_end.py
# The control: counts the lines of its input, reading until the input ends. Correct, and unbounded by design.
import sys

count = 0
line = sys.stdin.readline()
while line != "":
    count = count + 1
    line = sys.stdin.readline()
print(str(count) + " lines")
